src/daily_output.ts

```diff
@@ -192,6 +191,18 @@ export class DailyOutput {
 				tips: 0
 			};
 			this.data.ranks.lastWeek = this.data.ranks.weekly;
+			// updates the total rewards ok week change
+			this.data.ranks.weekly.groupEffortRewards.forEach(({ address, rewards }) => {
+				const prevTotal = this.data.ranks.total.groupEffortRewards.find(
+					({ address: addr }) => addr === address
+				);
+				if (prevTotal) {
+					const indexOfAddress = this.data.ranks.total.groupEffortRewards.indexOf(prevTotal);
+					this.data.ranks.total.groupEffortRewards[indexOfAddress].rewards += rewards;
+				} else {
+					this.data.ranks.total.groupEffortRewards.push({ address, rewards, rankPosition: 0 });
+				}
+			});
```

Comment:
Sort the total as well
```suggestion
			}).sort(({ address: address_1 }, { address: address_2 }) => weeklyTiebreakerSortFactory(this.data.wallets)(address_1, address_2));
```

Reply:
5e4c15390122c38d72c6a32625ba53938c12a044